cpp/CMakeLists.txt

```diff
@@ -294,6 +294,13 @@ else()
     set(NCCL_LIBRARIES ${NCCL_PATH}/lib/libnccl.so)
 endif(NOT NCCL_PATH)
 
+# - FAISS
+add_library(FAISS::FAISS SHARED IMPORTED)
+set_property(TARGET FAISS::FAISS PROPERTY
+	IMPORTED_LOCATION $ENV{CONDA_PREFIX}/lib/libfaiss.so)
```

Comment:
I'm worried that we'll need to support users that aren't using conda as well.  Is there another option we can use to find Faiss that don't rely on a `CONDA_PREFIX`, or is this the best way?

Reply:
Yes actually I added an option to build faiss statically on the [RAFT PR](https://github.com/rapidsai/raft/pull/126). I'm planning to do the same for cugraph.
https://github.com/hlinsen/raft/blob/tsp-knn/cpp/CMakeLists.txt#L180-L190.

Reply:
Updated this here: https://github.com/rapidsai/cugraph/pull/1360/commits/a3bb5ad769e3ca3342afd8f469e5574bbb951b2b